pyqtorch/core/__init__.py

```diff
@@ -10,4 +10,4 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
\ No newline at end of file
+# limitations under the License.
```

Comment:
@awennersteen It might be a good idea to import all the available operations (batched and not) to this `__init__.py` file to simplify importing.